### ISSUE TITLE
New Research Software Engineer @GESIS

### ISSUE THREAD

**Issue by arnim** [jupyterhub/team-compass#260]
Hi all,

we (GESIS, @bitnik and I) are searching for a Research Software Engineer
https://discourse.jupyter.org/t/gesis-is-searching-for-a-research-software-engineer-binder-jupyterhub/3501

That our administration greenlighted it today came as a surprise to me as well but I think this is a great opportunity to do some long(er) term planning. I would have loved to get your feedback on how we can make the best out of this. 

My current feeling is that we should strengthen subdomain.mybinder.org (including gesis.mybinder.org - I will try to do some convincing on your side) and develop a sustainable persistent offering that can be easily deployed and used by anyone how who wishes to do so.

What are your thoughts?
Best
Arnim

**Comment by meeseeksmachine** [comment]
This issue has been mentioned on **Jupyter Community Forum**. There might be relevant details there:

https://discourse.jupyter.org/t/gesis-is-searching-for-a-research-software-engineer-binder-jupyterhub/3501/2

**Comment by arnim** [comment]
One other thing that might be interesting to do is to integrate @betatim's **awesome bar** idea with some search- browsability of https://archive.analytics.mybinder.org

**Comment by choldgraf** [comment]
There are certainly many things that could be improved, exciting to hear that a new person will be around to help out! Thanks @arnim for keeping us in the loop, and happy to brainstorm ideas to work on

**Comment by arnim** [comment]
I'm very happy to announce that **@MridulS has joined us at GESIS Notebooks**. :rocket: :tada: 

@bitnik, @MridulS, and I will be at the upcoming Team Meeting #340

**Comment by betatim** [comment]
Super cool!

**Comment by arnim** [comment]
Btw .. is there anything we should do from the side of mybinder fed with respect to the transition from @bitnik to @MridulS e.g. jupyterhub/mybinder.org-deploy/pull/1698

**Comment by betatim** [comment]
I don't think so. @mriduls we often talk about mybinder.org deployment things in https://gitter.im/jupyterhub/mybinder.org-deploy and might mention you there. Otherwise let's figure it out as we go.

**Comment by sgibson91** [comment]
Has @MridulS been added to the binder google email group? Just been reminded as I saw we had an email from someone asking to have an image removed from GESIS' Docker Hub account.

**Comment by betatim** [comment]
Invite sent.

**Comment by arnim** [comment]
Hi all,
 
@MridulS and I were brainstorming today (besides network issues ;) on collaborating/integrating best between the federation (potential also 2i2c) and the offering we have at GESIS. One of the main questions (besides what's technically interesting to implement) is IMO how to best position the persistent offering without confusing users. 
 
I'd love to hear your input and advice on this. Do you (@sgibson91, @choldgraf, @betatim ...) think you would have a few minutes to brainstorm ... maybe right after the Feb Team Meeting?
 
I think this could really make a difference for us.

**Comment by sgibson91** [comment]
I'm on holiday when the next team meeting happens, but I'm very happy to leave thoughts/comments in a doc 🙂
